themes/grass/static/plugins/cookies/track.js

```diff
@@ -0,0 +1,20 @@
+export default function track() {
```

Comment:
we could use what they provide: https://developer.matomo.org/guides/tracking-javascript-guide :
replace what is in this file with the stuff they provide

````
  var _paq = window._paq = window._paq || [];
  _paq.push(['trackPageView']);
  _paq.push(['enableLinkTracking']);
  (function() {
    var u="//{$MATOMO_URL}/";
    _paq.push(['setTrackerUrl', u+'matomo.php']);
    _paq.push(['setSiteId', {$IDSITE}]);
    var d=document, g=d.createElement('script'), s=d.getElementsByTagName('script')[0];
    g.type='text/javascript'; g.async=true; g.src=u+'matomo.js'; s.parentNode.insertBefore(g,s);
  })();
  ```
  
  alternatively, put back this in the header: https://github.com/OSGeo/grass-website/commit/e48c0a1b8eb97138145968d1d1fc54a3f62f1a60